README.md

```diff
@@ -80,6 +80,21 @@ spec:
     #   Enable Kubernetes monitoring functionality
     #
     enabled: true
+
+  # Enables and configures infrastructure monitoring
+  # Collects data such as CPU or memory usage of host nodes.
+  #
+  infraMonitoring:
```

Comment:
Please include minimum Operator version, or to be released. Customers will look at this file once merged, regardless of Operator support.

Reply:
added "To be released"

Reply:
Please add 0.3.0 instead ;)